.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,68 @@
+name: 🐞 Bug
+description: Create a report to help us improve
+title: "<title>"
+labels: ["Bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Version
+      placeholder: |
+        $ kamu version
+        appVersion: 0.150.0
+        buildTimestamp: 2023-12-29T10:41:15.288818966Z
+        ...
+      render: sh
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: sh
+    validations:
+      required: true
```

Comment:
Logs are probably best attached as files - can we do that?

We should also be more specific and ask for `.kamu/run/kamu.log` **right after the execution of faulty command** (as we clean up the logs on start each time)

Reply:
Probably shouldn't be required also as some info in logs may be sensitve.

Reply:
> Logs are probably best attached as files - can we do that?

Yes, this input area is regular Github comment area, like the following 
![image](https://github.com/kamu-data/kamu-cli/assets/29308899/dbb47bff-6443-449b-932b-6b7611a5cb21)


---
Fixed via b32cb992db332fc98c9bda3cdc508e092d396f5e